docs/azure_jumpstart_arcbox/_index.md

```diff
@@ -0,0 +1,159 @@
+
+* Create Azure service principal (SP)
+
+    To deploy ArcBox an Azure service principal assigned with the "Contributor" role is required. To create it login to your Azure account run the below command (this can also be done in [Azure Cloud Shell](https://shell.azure.com/)).
+
+    ```shell
+    az login
+    az ad sp create-for-rbac -n "<Unique SP Name>" --role contributor
+    ```
+
+    For example:
+
+    ```shell
+    az ad sp create-for-rbac -n "http://AzureArcBox" --role contributor
+    ```
+
+    Output should look like this:
+
+    ```json
+    {
+    "appId": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "displayName": "AzureArcBox",
+    "name": "http://AzureArcBox",
+    "password": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "tenant": "XXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+    }
+    ```
+
+    > **Note: It is optional but highly recommended to scope the SP to a specific [Azure subscription and resource group](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-latest)**
+
+* Login to AZ CLI using the ```az login``` command.
+
+* Now you will need to edit the [azuredeploy.parameters.json](../../azure_jumpstart_arcbox/azuredeploy.parameters.json) ARM template parameters file and supply some values for your environment.
+
+  * *aksDnsPrefix* - AKS unique DNS prefix
+  * *sshRSAPublicKey* - Your SSH public key
+  * *spnClientId* - Your Azure service principal id
+  * *spnClientSecret* - Your Azure service principal secret
+  * *spnTenantId* - Your Azure tenant id
+  * *windowsAdminUsername* - Client Windows VM Administrator name
+  * *windowsAdminPassword* - Client Windows VM administrator password
+  * *myIpAddress* - Your local IP address. This is used to allow remote RDP and SSH connections to the Client Windows VM and K3s Rancher VM.
+  * *logAnalyticsWorkspaceName* - Unique name for the ArcBox log analytics workspace
+
+    ![Screenshot showing example parameters](./parameters.png)
+
+* Now you will deploy the ARM template. Navigate to the local cloned [deployment folder](../../azure_jumpstart_arcbox) and run the below command:
+
+  ```shell
+  az group create --name <Name of the Azure resource group> --location <Azure Region>
+  az deployment group create \
+  --resource-group <Name of the Azure resource group> \
+  --template-file azuredeploy.json \
+  --parameters azuredeploy.parameters.json 
+  ```
+
+  ![Screenshot showing az group create](./azgroupcreate.png)
+
+  ![Screenshot showing az deployment group create](./azdeploy.png)
+
+* After deployment, you should see the ArcBox resources inside your resource group.
+
+  ![Screenshot showing az deployment group create](./deployedresources.png)
+
+* Open a remote desktop connection into ArcBox-Client. Upon logging in, multiple automated scripts will open and start running. These scripts will take 10-20 minutes to finish. Once completed, Azure Data Studio will start. At this point, the deployment is complete.
+
+  ![Screenshot showing ArcBox-Client](./clientscript.png)
+
```

Comment:
I think there is a "Usage" section missing to show how to use the environment:

- Details on how to login the nested VM
- SQL Management studio in the SQL VM
- SQL Assessment in the portal
- No mention on kubectx and how to interact with the AKS/K3s clusters
- Azure Monitor for containers on the K3s Arc cluster (And AKS for the matter)
- azdata login to the controller
- etc